content/docs/user-guide/gems/repositories/overview.md

```diff
@@ -0,0 +1,194 @@
+
+{{< note >}}
+For a full description of `gem.json` fields and examples see the [`gem.json` manifest documentation](/docs/user-guide/programming/gems/manifest/).
+{{< /note >}}
+## Creating a Gem repository using a template 
+
+1. Create a new Gem repository using `o3de.bat`.  The following example uses a localhost URL as the repository URI for testing.
+
+    ```
+    scripts\o3de.bat create-from-template -tp Templates\GemRepo -dp <repository path on disk> -r ${RepoURI} http://localhost:8080/
+    ```
+
+1. Open the `repo.json` file in a text editor and update all the fields except the Gems list.  In the next step, you will update that when adding a Gem to the repository.
+    ```json
+    {
+        "repo_name":"RepositoryName",
+        "origin":"RemoteOriginName",
+        "repo_uri": "http://localhost:8080/",
+        "summary": "Example Repository",
+        "additional_info": "Example repository hosted on local http server.",
+        "last_updated": "10/26/2021 04:56pm",
+        "gems": [
+        ]
+    }
+    ```
+
+### Add a Gem to the Gem repository
+1.  Make a folder inside your Gem repository for the Gem.  In this example, the Gem is named `RemoteExample`. Make a folder named `RemoteExampleV1` and create a `gem.json` file in the folder that looks like this:
+    ```json
+    {
+        "gem_name": "RemoteExample",
+        "display_name": "Example Remote Gem",
+        "license": "Apache-2.0 Or MIT",
+        "origin":"RemoteOriginName",
+        "origin_uri": "http://localhost:8080/RemoteExampleV1/RemoteExampleV1.zip",
+        "sha256":"",
+        "type": "Code",
+        "summary": "An example remote gem.",
+        "canonical_tags": [
+            "Gem"
+        ],
+        "user_tags": [
+            "Gameplay"
+        ],
+        "icon_path": "preview.png",
+        "requirements": "",
+        "documentation_url": "https://o3de.org/docs",
+        "dependencies": []
+    }
+    ```
+
+    The `origin_uri` points to the location of the Gem's `.zip` archive that will be downloaded.
+
+1. Create a Gem, or take an existing Gem, zip up the Gem's contents, not the folder itself, and place the `.zip` in the "RemoteExampleV1" folder, and name it `RemoteExampleV1.zip`.
+{{< tip >}}
+When creating an archive of your Gem, zip up the contents of the Gem folder, not the folder itself, so that when extracted, the `gem.json` file will be at the root, and not inside a subfolder named `RemoteExampleV1` 
+{{< /tip >}}
+1. Generate the SHA-256 for your Gem's `.zip` archive and add that value to the `sha256` field of the `gem.json` file located outside the `.zip`.  On Windows, you can use the `certutil` program to generate the SHA-256 value like this:
+{{< note >}}
+It is possible, but not recommended, to leave out the SHA-256 field for testing.
+{{< /note >}}
+    ```
+    certutil -hashfile C:/path/to/gem.zip SHA256
+    ```
```

Comment:
```suggestion
    ```
    certutil -hashfile C:/path/to/gem.zip SHA256
    ```
    {{< note >}}
It is possible, but not recommended, to leave out the SHA-256 field for testing.
{{< /note >}}
```
Copied from a resolved conversation:

The opening tag is indented the expected amount (here, four spaces, to match the indentation of the code block). The indentation level of the tag's body is what causes issues. Any indentation in a tag's body is treated as a new level. Indenting the body to match the opening tag will create something that looks like a codeblock inside of the note. Instead, remove all indentation from the body (the closing tag's indentation level doesn't matter). I'll suggest this exact change for clarity.

Reply:
Excellent, not indenting the rest of the tag contents worked.